CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 22.7.5
 
+### Breaking Changes
+- Flexible Privacy Groups support to Tessera's EC encryptor (contracts modified) [#4282](https://github.com/hyperledger/besu/pull/4282) 
+  * Before this change, the `bytes32` type was used for the enclave public keys, just supporting encryptors with public keys of that length (like the default NaCl)
+  * For the EC encryptor, the encoded public key length is 91
```

Comment:
needs to move to 22.10.0-RC1

Reply:
and maybe mention that it's still early access - Flexible Privacy Groups (early access)

Reply:
Done, thanks @macfarla !